src/data/finsupp/basic.lean

```diff
@@ -1825,6 +1829,26 @@ begin
     apply h_cases (hl $ finset.mem_coe.2 $ mem_support_iff.2 $ λ h, h_contr h.symm) }
 end
 
+variable {f}
+
```

Comment:
Can you make v₁ and v₂ explicit here?

Reply:
Done.  In doing so, I also noticed that they appeared in what I think was reverse order, so I reordered them.

Should `f` be implicit in the previous lemma as well?  I made it implicit in the new lemmas, but I sometimes regret these decisions.

Reply:
`f` is inferred from `hf` in the new lemmas so I think that's ok

Reply:
It is inferred also in the old one `map_domain_comap_domain`: should that one also have an implicit `f`?